galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/spi/language/gherkin/GherkinVariables.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
```

Comment:
Should have the EPL statement in also. Once copyright checker is back it would show this error up.